playbook/app/pb_kits/playbook/pb_currency/currency.html.erb

```diff
@@ -1,17 +1,24 @@
 <%= pb_content_tag do %>
   <%= pb_rails("caption", props: object.caption_props) %>
-
   <div class=<%= "pb_currency_wrapper#{object.variant_class || object.emphasized_class}" %>>
     <% if object.unstyled %>
-      <div><%= object.negative_sign %><%= object.symbol %></div>
-      <div><%= object.title_props[:text] %></div>
-      <div><%= object.body_props[:text] %></div>
+      <% if object.null_display && object.amount.blank? %>
+        <div><%= object.null_display %></div>
+      <% else %>
+        <div><%= object.negative_sign %><%= object.symbol %></div>
+        <div><%= object.title_props[:text] %></div>
+        <div><%= object.body_props[:text] %></div>
+      <% end %>
     <% else %>
-      <%= pb_rails("body", props: object.currency_wrapper_props) do %>
-        <%= object.negative_sign %><%= object.symbol %>
+     <% if object.null_display && object.amount.blank? %>
+        <%= pb_rails("title", props: {text: object.null_display}) %>
```

Comment:
Using title kit here is causing the null value to always be displayed as a title, which may not be what we want!

In addition to this, do you think we need to account for emphasized variant, light and bold variants? If they are using this null as a value within a table, my assumption is they would want consistency in the display, so if the rest of the values are light, they'll want this to be light? 

We can also always only account for whatever the default is for this kit and add follow ups to account for different variants. I would also reach out to a designer with these questions!

Reply:
@nidaqg great call!! What should we have here instead?